<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Authorization3</class>
 <widget class="QWidget" name="Authorization3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>675</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Вход</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{background-color: rgb(47, 47, 47);}</string>
  </property>
  <widget class="QLabel" name="login_text">
   <property name="geometry">
    <rect>
     <x>417</x>
     <y>203</y>
     <width>64</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{font-size: 24px; color: rgb(255, 255, 255);}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Login&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_email">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>328</y>
     <width>370</width>
     <height>35</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="whatsThis">
    <string>Input your email</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{background-color: rgb(41, 90, 109); color: rgb(255, 255, 255); border-style: solid; border-width:0px; border-radius: 15px; border-color:rgb(41, 90, 109); max-width:370px; max-height:35px; min-width:370px; min-height: 35px}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="text_email">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>300</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{font-size: 14px; color: rgb(255, 255, 255)}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Email&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="text_password">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>375</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{font-size: 14px; color: rgb(255, 255, 255)}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Password&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="to_forgot_password">
   <property name="geometry">
    <rect>
     <x>545</x>
     <y>450</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color: rgb(47, 47, 47); color: rgb(255, 255, 255); font-size: 10px;border-width:0px; border-radius: 0px}</string>
   </property>
   <property name="text">
    <string>Forgot password?</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_password">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>403</y>
     <width>370</width>
     <height>35</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>Input your password</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{background-color: rgb(41, 90, 109); color: rgb(255, 255, 255); border-style: solid; border-width:0px; border-radius: 15px; border-color:rgb(41, 90, 109); max-width:370px; max-height:35px; min-width:370px; min-height: 35px}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="to_sign_up">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>570</y>
     <width>182</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color: rgb(47, 47, 47); color: rgb(255, 255, 255); font-size: 13,79px;border-width:0px; border-radius: 0px}</string>
   </property>
   <property name="text">
    <string>Don't have an account? Sign up</string>
   </property>
  </widget>
  <widget class="QPushButton" name="to_login">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>530</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color:  rgb(255, 255, 255); color: rgb(47, 47, 47); font-size: 14px; font-weight:600; border-radius: 15px}
</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
